test/Microsoft.AspNetCore.Razor.Language.Test/CodeGeneration/RedirectedRuntimeBasicWriterTest.cs

```diff
@@ -207,7 +207,7 @@ public void WriteHtmlContent_RendersContentCorrectly()
             // Assert
             var csharp = context.Writer.Builder.ToString();
             Assert.Equal(
-@"WriteLiteralTo(test_writer, ""SomeContent"");
+@"WriteLiteralTo(test_writer, @""SomeContent"");
```

Comment:
The `RedirectedRuntimeBasicWriter` class is derived from `RuntimeBasicWriter`. And so `WriteLiteral` is replaced by `WriteLiteralTo`, but with the same tokens.